src/src/DataStructs/NotificationSettingsStruct.h

```diff
@@ -9,7 +9,7 @@
 
 # define NPLUGIN_001_DEF_TM     8000  // Email Server Default Response Time, in mS.
 # define NPLUGIN_001_MIN_TM     5000
-# define NPLUGIN_001_MAX_TM     20000
+# define NPLUGIN_001_MAX_TM     12000
```

Comment:
The relative difference between 12 and 20 seconds is rather small, so I think 20 seconds is not a bad maximum to have (server responses might become even worse... 😟) . Just my 2 🪙

Reply:
Easy to change. Let's see how TD-er feels about this.

Reply:
I truly wonder what may cause these enormous delays thus why it is needed?

Reply:
I believe the old timeout (5000mS) was reliable, but hard to know for sure since I only send ESPEasy emails when a device does a random reboot. 

The timeout issue is just one of the problems that I uncovered after my host account's ISP server updates broke ESPEasy's ability to send emails.

Keep in mind that I have to use smtp2go as a SSL bridge for ESPEasy email. So there is a man-in-the-middle situation. So hard to say which part is to blame for the sluggish response.

Reply:
Hmm there might also be another issue at hand here...
If you are using LittleFS builds, then there might be some other 'fix' in the Arduino code which does break a lot.
In the latest code/testbuilds this merged commit has been reverted.

If you are using LittleFS builds, then maybe you can also try changing this in `platformio_core_defs.ini` 

```ini
[core_esp32_IDF5_1__3_0_0]
platform                    = https://github.com/tasmota/platform-espressif32/releases/download/2024.06.11/platform-espressif32.zip
platform_packages           = framework-arduinoespressif32 @ https://github.com/Jason2866/esp32-arduino-lib-builder/releases/download/2543/framework-arduinoespressif32-all-release_v5.1-bd03553.zip
```

So only change those 2 lines with `platform` and `platform_packages` in the `[core_esp32_IDF5_1__3_0_0]` section (at the bottom of that file)

Reply:
That's interesting.
Build log shows I am compiling with File System: SPIFFS
It's based on ESP_Easy_mega_20240702_normal_ESP32_4M316k.

Reply:
OK, then that's not the 'bug' that may be causing this.